.github/workflows/cd.yml

```
@@ -15,9 +15,18 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: DeterminateSystems/nix-installer-action@main
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)